DEVELOPER.md

```diff
@@ -37,7 +37,7 @@ and the most important variables, run `make help`, e.g:
 VARIABLES:
   APPS          = config configsubscription firehose forwarder logwriter metricstream stack
   AWS_REGION    = us-west-2
-  GO_BINS       = forwarder subscriber
+  GO_BINS       = forwarder subscriber metricsconfigurer
```

Comment:
i think configurer and configurator both work here, i just feel like configurator sounds better/is easier to say haha